compose/cli/main.py

```diff
@@ -232,13 +237,13 @@ def pull(self, project, options):
         Usage: pull [options] [SERVICE...]
 
         Options:
-            --allow-insecure-ssl    Allow insecure connections to the docker
-                                    registry
+            --allow-insecure-ssl    Deprecated - no effect.
```

Comment:
I think it's better to keep it. Removing it immediately would break any scripts which include this flag. Leaving it here as a no-op gives people more time to remove it without having it break on upgrade.